app/controllers/application_controller.rb

```diff
@@ -47,6 +47,10 @@ def decorated_session
     @_decorated_session ||= DecoratedSession.new(sp: current_sp, view_context: view_context).call
   end
 
+  def signed_in_path
```

Comment:
It needs to be a public method so it can be tested. Alternatively, we can make it a private method and only rely on integration tests.